# Input vector.
float64[] input_vec

# Output vector.
float64[] output_vec